<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserSettingsWindow</class>
 <widget class="QWidget" name="UserSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnReturn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>510</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>211</width>
     <height>212</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="6" column="0">
     <widget class="QPushButton" name="btnEnter">
      <property name="text">
       <string>Потвердить</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="OldPassword_2">
      <property name="text">
       <string>Старый пароль</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="NewPassword">
      <property name="text">
       <string>Новый пароль</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="RepeatPassword">
      <property name="text">
       <string>Повторите пароль</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="leNewPassword_2"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="leOldPassword_2"/>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="leRepeatPassword_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Settings">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Настройки пользователя</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Market Mentor</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
